fastdupes.py

```diff
@@ -495,7 +495,7 @@ def print_defaults():
         value = DEFAULTS[key]
         if isinstance(value, (list, set)):
             value = ', '.join(value)
-        print "%*s: %s" % (maxlen, key, value)
```

Comment:
This *completely* changes the behaviour of the maxlen argument.

The `%*s` means "Format `key`, padding it out to `maxlen`". What you changed it to means "Print `maxlen`, a space, and then `key`".

The fact that you clearly haven't tested these changes makes me wary of accepting the pull request overall, when the whole reason I didn't do this is that I wouldn't have time to test them.

(Switching to Python 3 syntax is quick and easy. Thoroughly testing the changes is the time-consuming bit.)